### ISSUE TITLE
Settings Crash

### ISSUE THREAD

**Issue by realbillie** [EamonTracey/SecretShot#4]
Can't enable tweak because when clicking SecretShot in setting, crashes settings app to Home Screen. Any updates? Please?!

**Comment by RipGlock** [comment]
I believe this is a compiling error. Im assuming you are ARM64e? Im trying to build the file right now i will update you if i do.

**Comment by realbillie** [comment]
Yes, I am on IPhone 12 Pro Max 14.4 jailbroken with unc0ver. Any updates lately? Keep me informed please

**Comment by eternal404** [comment]
duplicate of #1

**Comment by surtaxkurd** [comment]
> I believe this is a compiling error. Im assuming you are ARM64e? Im trying to build the file right now i will update you if i do.

Any update on the tweak? It still crashes everytime I try to activate it. Can you please resolve this issue. I’m on IOS 14.4

**Comment by marioparaschiv** [comment]
This project was abandoned from the looks of it, you may use https://github.com/localip/phantom if you want something that works.

**Comment by surtaxkurd** [comment]
> This project was abandoned from the looks of it, you may use https://github.com/localip/phantom if you want something that works.

Thank you so much! You saved me. Working like charm.
